lib/collection/src/shards/replica_set/update.rs

```diff
@@ -71,6 +71,7 @@ impl ShardReplicaSet {
         operation: CollectionUpdateOperations,
         wait: bool,
         ordering: WriteOrdering,
+        update_existing: bool,
```

Comment:
I was a bit confused by this name when I first came across it, especially because we pass it along a few times.

Would `only_update_existing` be a bit better?

I couldn't really come up with any other good name, and don't like `ignore_non_exists_errors`.

Reply:
We could not come up with a unanimously better name, so we decided to leave it as is for now. :/